speech.js

```diff
@@ -134,41 +192,194 @@ class Speech {
     this.apiKey = apiKey;
   }
 
-  async fetchVoices() {
-    return fetch(URL_VOICES, {
+  async fetchVoices(options = {}) {
+    const starred = options.starred || false;
+    const owner = options.owner || 'all';
+    let url = `${_BASE_URL}${_LIST_VOICES_ENDPOINT}?starred=${starred}&owner=${owner}`;
+    return fetch(url, {
+      headers: this._getHeaders(),
+      method: "GET",
+    })
+      .then(async (response) => {
```

Comment:
I think you can do this instead of the `async` and `await`, both here and at line 218, 288, 303, 332, 362 below:

```
.then(response => {
    return this._handle_response_errors(response)
      .then(() => response.json());
  })
```